modules/frontend/manifests/packages.pp

```diff
@@ -8,14 +8,14 @@
   exec { "nodepackages":
     path    => "/bin:/usr/bin",
     unless  => "npm list -g --depth=0 | grep grunt-cli",
-    command => "npm install -g coffee-script bower grunt-cli gulp component yo",
```

Comment:
Remove 'em all! https://github.com/kalamuna/kalastack/pull/111

Installing things globally is good, but it's messing things up. We've moved to installing things locally in `package.json` on the application layer.